keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -10,8 +10,8 @@
   - [Non-Goals](#non-goals)
 - [Proposal](#proposal)
   - [Implementation](#implementation)
-    - [On using pvc.Spec.AllocatedResources](#on-using-pvcspecallocatedresources)
     - [User flow stories](#user-flow-stories)
+      - [Case 0 (default PVC creation):](#case-0-default-pvc-creation)
```

Comment:
This looks weird, where are the other cases? IMO you have wrong nr. of `#` in `Case 0`